langfuse/src/langfuse.ts

```diff
@@ -8,6 +8,9 @@ import {
 import { type LangfuseStorage, getStorage } from "./storage";
 import { version } from "../package.json";
 import { type LangfuseOptions } from "./types";
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
Will this here work in all the environments like Browsers or different Node versions?

Reply:
i test this on node version
    18.18.0
    18.13.0
    16.14.2
    16.0.0 
    14.17.0
    12.11.1
it works perfectly